README.md

```diff
@@ -24,34 +27,65 @@ Backend
 
   <img src="https://img.shields.io/badge/django%20-%23092E20.svg?&style=for-the-badge&logo=django&logoColor=white"/>   <img src="https://img.shields.io/badge/sqlite-%2307405e.svg?&style=for-the-badge&logo=sqlite&logoColor=white"/>
 
-## ⚙Installation
+# ⚙Installation
 
-1. Star the Repo 🌟 and this will keep me motivated.
+<p> Star the Repo 🌟 and this will keep me motivated. </p>
 
-2. Install django and requests by typing the below command in terminal.
+1. Download and install Python
+2. Download and install Git.
+3. Fork the Repository.
+4. Clone the repository to your local machine 
 ```sh
-pip install -r requirements.txt
+git clone https://github.com/<your-github-username>/Rescue.git
 ```
 
-3. Fill the details in  
-```
-main_app/mail.py
-mysite/settings.py
+5. Change directory to main_app
+```sh
+cd main_app
 ```
 
-4. Run the website by following command in the terminal
+6.. Install django and other requirements by using the command below.
 ```sh
+pip install -r requirements.txt
+```
+
+4. Fill the details in the files: 
+  
+  * [main_app/mail.py](main_app/mail.py)
+
+  * [mysite/settings.py](main_app/settings.py)
+
+5. Make migrations
+```bash
 python manage.py makemigrations
+```
+
+6. Migrate the changes to the database
+```bash 
 python manage.py migrate
+```
+
+7. Create admin 
+```bash
 python manage.py createsuperuser
+```
+
+8. Run the server
+```bash
 python manage.py runserver
 ```
-## First time to [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
-Read [how to contribute.](https://github.com/swapnilsparsh/Rescue/blob/master/Contributing.md)
+9. View the website by pasting the address in your browser.
+```bash
+https://localhost:8000
```

Comment:
Mention that you can open the website by clicking on the link i.e there on the terminal also by adding or option.

Reply:
The terminal one is to copy and paste. I'll make it a link.